.github/workflows/pull_request_labeler.yml

```diff
@@ -0,0 +1,19 @@
+name: "Pull Request Labeler"
+on:
+  pull_request: # Temporary, for testing while PR is still open
+  # pull_request_target
```

Comment:
Note to reviewers: please don't merge this PR with this code in. Before merging, we need to delete `pull_request` and uncomment `pull_request_target`.

See https://github.com/actions/labeler#notes-regarding-pull_request_target-event for more details